sql: add spaces to delimiter in uniqueness violation error

michae2 opened the pull request:
Whenever a mutation statement violates a uniqueness constraint we return a specific error which shows the affected row. This error is formatted to match the corresponding error in Postgres.

We create this error in two places. In one place (`pkg/sql/opt/exec/execbuilder/mutation.go`) we were using ", " as the delimter between values, which matches Postgres. But in the other place (`pkg/sql/row/errors.go`) we were using "," (without a space).

This patch adds the space.

Epic: None

Release note (bug fix): Fix formatting of uniqueness violation errors to match the corresponding errors from PostgreSQL.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/96914)
<!-- Reviewable:end -->

michae2 committed:
sql: add spaces to delimiter in uniqueness violation error

Whenever a mutation statement violates a uniqueness constraint we return
a specific error which shows the affected row. This error is formatted
to match the corresponding error in Postgres.

We create this error in two places. In one place
(`pkg/sql/opt/exec/execbuilder/mutation.go`) we were using ", " as the
delimter between values, which matches Postgres. But in the other place
(`pkg/sql/row/errors.go`) we were using "," (without a space).

This patch adds the space.

Epic: None

Release note (bug fix): Fix formatting of uniqueness violation errors to
match the corresponding errors from PostgreSQL.

rharding6373 reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/96914#-:-NOuIDPX7WjEbKePJ5Ji:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

michae2 commented:
Thanks!

bors r=rharding6373
